test/spec.js

```diff
@@ -38,22 +38,25 @@ describe('spec', function() {
 
         if (exists(t.src)) {
           it(test, function(done) {
-            var expected = util.normalize(read(t.expected, 'utf8'));
-
-            sass.render({
-              file: t.src,
-              includePaths: t.paths
-            }, function(error, result) {
-              if (t.error) {
-                assert(error);
-              } else {
-                assert(!error);
-              }
-              if (expected) {
-                assert.equal(util.normalize(result.css.toString()), expected);
-              }
-              done();
-            });
+            // var expected = util.normalize(read(t.expected, 'utf8'));
+            assert(true);
+            done();
+            return;
+            // sass.render({
```

Comment:
Why do we need to remove this test?

Reply:
sorry, I'm testing the issue with the windows, when process doesn't exist when running `npm test`